packages/scripts/src/components/CodeEditor.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <div>
+    <div class="position-relative">
+      <form-component v-if="showField"
+                      id="CodeEditor"
+                      :formFields="codeEditorFields"
+                      :initialFormData="codeEditorData"
+                      :formState="codeEditorState"
+                      :options="options"
+                      @valueChange="onValueChanged"></form-component>
+      <span id="in-corner"><slot></slot></span>
```

Comment:
i do not understand this id

Reply:
Changed to class, used to target and place button via css